runtime/common/src/precompile/multicurrency.rs

```diff
@@ -56,12 +59,14 @@ pub enum Action {
 	QueryTotalIssuance = "totalSupply()",
 	QueryBalance = "balanceOf(address)",
 	Transfer = "transfer(address,address,uint256)",
+	Transfer2 = "transfer2(address,bytes32,uint256)",
```

Comment:
can we have a better name?

Reply:
do you have a suggestion? also this is just kinda following the naming convention 💁‍♂️ like create & create2

Reply:
TransferToAccountId, TransferNativeDestination